src/containers/Header/component.jsx

```diff
@@ -22,6 +24,9 @@ const Header = (props) => {
     </div>
     <div className={styles.Header_logo} />
     <div className={styles.Header_wrap}>
+      {Config.get('notificationsTray') ?
+        <NotificationsTray /> :
+        null}
```

Comment:
It's a little more dangerous as `0` (or `false`, maybe) would get injected as text into the page